cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/ExecutionTaskManager.java

```diff
@@ -241,45 +273,75 @@ public void markTasksInProgress(List<ExecutionTask> tasks) {
   }
 
   /**
-   * Mark the successful completion of a given task. Only normal execution may yield successful completion.
+   * Mark the successful completion of a given task. In-progress execution may yield successful completion. 
+   * Aborting execution may yield Aborted completion.
```

Comment:
Why do we say `may yield`? Doesn't aborting execution always yield Aborted completion in this method?